src/main/java/org/folio/rest/impl/PurchaseOrderLineHelper.java

```diff
@@ -333,6 +341,49 @@ private String buildPoLineNumber(String poNumber, String sequence) {
     return poNumber + "-" + sequence;
   }
 
+  /**
+   * See MODORDERS-180 for more details.
+   * @param compPoLine composite PO Line
+   */
+  private void updateEstimatedPrice(CompositePoLine compPoLine) {
+    Cost cost = compPoLine.getCost();
+    if (cost != null) {
+      cost.setPoLineEstimatedPrice(calculateEstimatedPrice(cost));
+    }
+  }
+
+  /**
+   * Calculates total estimated price. See MODORDERS-180 for more details.
+   * @param cost PO Line's cost
+   */
+  double calculateEstimatedPrice(Cost cost) {
+    BigDecimal total = BigDecimal.ZERO;
+    if (cost.getListUnitPrice() != null) {
```

Comment:
I think it would be nice to add these validations in the PurchaseOrderHelper.validateOrder method too, so that we do not end put calculating electronic costs for physical orders

Reply:
Validation has been updated